docs/tutorial/electron-versioning.md

```diff
@@ -18,7 +18,7 @@ npm install --save-dev electron@latest
 
 There are several major changes from our 1.x strategy outlined below. Each change is intended to satisfy the needs and priorities of developers/maintainers and app developers.
 
-1. Strict use of the the [SemVer](#semver) spec
+1. Strict use of the [SemVer](#semver) spec
```

Comment:
Duplicate heuristics are a relatively new thing for me, I'm still not terribly confident about them...

Reply:
This change seems fine to me - the second "the" is unnecessary